README.md

```diff
@@ -49,6 +49,7 @@ as sufix. The bullets below explains what each column represent.
 * **status**: It gives the http status of a request. 200 means successfully request, 503 means failed requests.  
 * **done**: It gives the request state. True if the request was processed and False otherwise. 
```

Comment:
Remove this and the line bellow. These are not part of the output anymore.

Reply:
They are used when simulator generates a log results with more detailed information.